templates/zerver/help/delete-a-stream.md

```diff
@@ -3,8 +3,8 @@
 {!admin-only.md!}
 
 In Zulip, most stream administration is done on the subscription page.
-However, Zulip organization administrators must use the **Manage
-organization** interface to delete streams.
+However, Zulip organization administrators must use the
+**Organization** interface to delete streams.
```

Comment:
Both this and the previous one read oddly.  I think we should just say something like 

"Zulip [organization administration interface]/help/change-your-organization-settings() ..."